### PR TITLE
Added color() as a global alias of parse_color()

### LINKED ISSUE #54: Expose parse_color as a global function called "color"
Acceptance Criteria (AC):
 * [x] Expose parse_color as `color` (e.g. `color(r, g, b)`)
    * Should be compatible with any inputs that `parse_color` accepts (e.g. `color("red")`)
    * Should be a global immutable function
 * [x] Add unit tests

### PR DESCRIPTION
**Pull request by Hesham-E**

### CODE REVIEW

**Comment by lawruble13** [comment]
Previously, `color` did not return the result of `parse_color`. I added this, and another test to check that `color` can be used in the intended manner. Otherwise, good addition!

**Comment by lawruble13** [comment]
Note that current practice is for pull requests to go to  `dev`, not `main`, so that `main` isn't changing between versions, and to keep our flows operating.

**Comment by Hesham-E** [comment]
> Note that current practice is for pull requests to go to `dev`, not `main`, so that `main` isn't changing between versions, and to keep our flows operating.

Alright, thank you for letting me know! That makes sense!

File: spark/core.py [hunk 1]
```diff
@@ -431,6 +431,9 @@ def stroke_width(self, *args): pass
     @extern
     def parse_color(self, *args, func_name="parse_color"): pass
 
+    @extern
```
Comment (lawruble13) [inline spark/core.py:434]: Correctly imported.

File: spark/util/helper_functions/misc_functions.py [hunk 2]
```diff
@@ -50,6 +50,10 @@ def clip(x, lb, ub):
     else:
         raise ArgumentNumError(func_name, [1, 3, 4], argc)
 
+@validate_args([str], [Real], [Real, Real, Real], [Real, Real, Real, Real])
```
Comment (lawruble13) [inline spark/util/helper_functions/misc_functions.py:53]: Correctly validated, declared global, and defined with name.

**Review by lawruble13 — APPROVED** [review]

**Pull request merged**